airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql

```diff
@@ -0,0 +1,78 @@
+---
+operator: operators.SqlToWarehouseOperator
+dst_table_name: "views.gtfs_rt_extraction_html_errors"
```

Comment:
Do we want to call this `http_errors` rather than `html_errors`?

Reply:
ahh good call